teachers_digital_platform/crtool/src/js/components/CustomerReviewToolComponent.js

```diff
@@ -0,0 +1,163 @@
+    switch(changedDistinctive) {
+      case C.CONTENT_DISTINCTIVE:
+        localStorage.setItem(C.CONTENT_STATUS, distinctiveStatus);
+        this.setState({contentInProgress: distinctiveStatus});
+        break;
+      case C.UTILITY_DISTINCTIVE:
+        localStorage.setItem(C.UTILITY_STATUS, distinctiveStatus);
+        this.setState({utilityInProgress: distinctiveStatus});
+        break;
+      case C.QUALITY_DISTINCTIVE:
+        localStorage.setItem(C.QUALITY_STATUS, distinctiveStatus);
+        this.setState({qualityInProgress: distinctiveStatus});
+        break;
+      case C.EFFICACY_DISTINCTIVE:
+        localStorage.setItem(C.EFFICACY_STATUS, distinctiveStatus);
+        this.setState({efficacyInProgress: distinctiveStatus});
+        break;
+      default:
+        break;
+    }
+  }
+
+  distinctiveClicked(clickedDistinctive) {
+    localStorage.setItem(C.START_DISTINCTIVE, clickedDistinctive);
+    this.setState({currentPage: clickedDistinctive});
+  }
+
+  render() {
+
+    const distinctiveProps = [
+      {
+        title:"Content",
+        criteria:"6 criteria",
+        estimatedtime:"Est. time 30 min",
+        description:"Covers core knowledge and skills in content standards",
+        distinctive:C.CONTENT_DISTINCTIVE,
+        inProgress:this.state.contentInProgress,
+        distinctiveClicked:this.distinctiveClicked.bind(this),
+      },
+      {
+        title:"Utility",
+        criteria:"7 criteria",
+        estimatedtime:"Est. time 30 min",
+        description:"Supports effective teaching",
+        distinctive:C.UTILITY_DISTINCTIVE,
+        inProgress:this.state.utilityInProgress,
+        distinctiveClicked:this.distinctiveClicked.bind(this),
+      },
+      {
+        title:"Quality",
+        criteria:"5 criteria",
+        estimatedtime:"Est. time 30 min",
+        description:"Accurate and well presented",
+        distinctive:C.QUALITY_DISTINCTIVE,
+        inProgress:this.state.qualityInProgress,
+        distinctiveClicked:this.distinctiveClicked.bind(this),
+      },
+      {
+        title:"Efficacy",
+        criteria:"3 criteria",
+        estimatedtime:"Est. time 30 min",
+        description:"Improves financial knowledge, skills, or behaviors",
+        distinctive:C.EFFICACY_DISTINCTIVE,
+        inProgress:this.state.efficacyInProgress,
+        distinctiveClicked:this.distinctiveClicked.bind(this),
+      }
+    ]
+
+    return (
+      <div className="block block__flush-top">
+        <h1>Curriculum Review</h1>
+        <p>Middle School Example Curriculum</p>
+        <p>Start the review by selecting a dimension. You do not need to complete all dimensions in one sitting. You’ll be able to download a dimension report for each dimension as well as a summary report at the end.</p>
+        
+        <div className="DistinctivesBlock" >
+          {distinctiveProps.map(distinctiveProps => <DistinctiveButton distinctiveClicked={this.distinctiveClicked.bind(this)} {...distinctiveProps}/>)}
```

Comment:
You can drop the `distinctiveClicked={this.distinctiveClicked.bind(this)}` bit from here, as that's getting covered by the `distinctiveClicked` held in the props.

Reply:
Thanks for catching this...